lib/array.js

```diff
@@ -192,14 +204,17 @@ const series = (
   next();
 };
 
-const find = (
-  // Asynchronous find (iterate in series)
-  items, // array, incoming
-  fn, // (value, callback) => callback(err, accepted)
-  //   value - item from items array
-  //   callback - callback function(err, accepted)
-  done // function (optional), on done callback function(err, result)
-) => {
+// Asynchronous find (iterate in series)
+//   items - <Array>, incoming
+//   fn - <Function>,
+//     value - <any>, item from items array
+//     callback - <Function>
+//       err - <Error>
+//       accepted - <boolean>
+//   done - <Function>, on done, optional
+//     err - <Error>
+//     result - <Array>
```

Comment:
This may be any value, not just an array:
```suggestion
//     result - <any>
```